site/docs/v1/tech/apis/prometheus.adoc

```diff
@@ -0,0 +1,73 @@
+---
+layout: doc
+title: Prometheus API
+description: API for retrieving application metrics to be used with prometheus
+---
+
+
+[NOTE.since]
+====
+This API has been available since 1.26.0
+====
+
+
+== Overview
+
+This page contains the API that is used for retrieving FusionAuth application metrics to be used with Prometheus. Please refer to the link:/docs/v1/tech/tutorials/prometheus#[Prometheus setup] guide to understand how to setup Prometheus with the FusionAuth Prometheus endpoint.
+
+
+* <<Retrieve the Metrics>>
+
+
+== Retrieve the Metrics
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Retrieve FusionAuth application metrics to use with Prometheus.
+[.endpoint]
+.URI
+--
+[method]#GET# [uri]#/api/prometheus/metrics#
+--
+
+=== Authentication
+Prometheus does not provide functionality to authenticate using custom HTTP headers. As a workaround, we are using API key as password using "basic auth" authorization method. In order to authenticate using basic auth please use string "apikey" as the username and a valid API key as the password. The API key must have `GET` permission for the `/api/prometheus/metrics` endpoint.
+
+Please refer to the link:/docs/v1/tech/apis/authentication#basic-authentication-with-api-key[Authentication] document for more details about `basic auth` authentication method using API keys.
+
+=== Request Parameters
+There are no request parameters required with this API.
+
+
+=== Response
+The response to this API call is various metrics that are available. The following types of gauges are currently available.
```

Comment:
We may want to document, similar to the Status API that the response is subject to change.